README.md

```diff
@@ -1,9 +1,9 @@
 # Barefoot Nomad
 
-[![Build Status](https://travis-ci.com/atlp-rwanda/space-bn-frontend.svg?branch=develop)](https://travis-ci.com/atlp-rwanda/space-bn-frontend)
-<a href="https://codeclimate.com/github/atlp-rwanda/space-bn-frontend/maintainability"><img src="https://api.codeclimate.com/v1/badges/3e75b6bbdd43289b59ec/maintainability" /></a>
-<a href="https://codeclimate.com/github/atlp-rwanda/space-bn-frontend/test_coverage"><img src="https://api.codeclimate.com/v1/badges/3e75b6bbdd43289b59ec/test_coverage" /></a>
-[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/space-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/space-bn-frontend?branch=develop)
```

Comment:
We still also need coverage badge

Reply:
@SimonImanigirimpuhwe please confirm if your change request has been attended to and resolve

Reply:
the change is resolved